cdap-ui/app/features/hydrator/services/hydrator-service.js

```diff
@@ -81,6 +81,8 @@ class HydratorService {
     var params = {
       namespace: this.$state.params.namespace,
       pipelineType: this.ConfigStore.getAppType(),
+      // version: plugin.artifact.version || this.$rootScope.cdapVersion,
+      // TODO: Should this be plugin's artifact version? 
       version: this.$rootScope.cdapVersion,
```

Comment:
So right now backend returns a 404 for it. Will confirm with @albertshau before making this change.